apps/learner/src/lib/server/weaviate.ts

```diff
@@ -21,15 +21,44 @@ export const weaviate = await Weaviate.connectToCustom({
   skipInitChecks: true,
 });
 
+/**
+ * Performs hybrid search on the LearningUnit collection.
+ *
+ * Uses both keyword and vector similarity search to find relevant learning content.
+ * The search targets the 'content' property and uses the 'content_vector' for
+ * semantic similarity matching.
+ *
+ * @param query - The search query string
+ * @returns Promise<string[]> - Array of content strings from matching learning units
+ *
+ * @example
+ * ```typescript
+ * const results = await search("machine learning algorithms");
+ * console.log(results); // ["Content about ML algorithms...", "Another related content..."]
+ * ```
```

Comment:
```suggestion
 * Search for relevant learning content by using both keyword and vector similarity.
 *
 * @param query - The query to search for.
 * @returns A list of relevant learning content.
 *
 * @example
 * ```ts
 * const results = await search('machine learning algorithms');
 * console.log(results); // ["Content about ML algorithms...", "Another related content..."]
 * ```
```